pkg/metrics/metrics.go

```diff
@@ -275,13 +275,14 @@ func (m *MetricsConfig) RecordPolicyRuleInfo(policyValidationMode PolicyValidati
 	m.policyRuleInfoMetric.Observe(ctx, metricValue, commonLabels...)
 }
 
-func (m *MetricsConfig) RecordAdmissionRequests(resourceKind string, resourceNamespace string, resourceRequestOperation ResourceRequestOperation) {
+func (m *MetricsConfig) RecordAdmissionRequests(resourceKind string, resourceNamespace string, resourceRequestOperation ResourceRequestOperation, allowed bool) {
 	ctx := context.Background()
 
 	commonLabels := []attribute.KeyValue{
 		attribute.String("resource_kind", resourceKind),
 		attribute.String("resource_namespace", resourceNamespace),
 		attribute.String("resource_request_operation", string(resourceRequestOperation)),
+		attribute.Bool("request_allowed", allowed),
```

Comment:
Shall it be `resource_request_allowed` ?

Reply:
`request_allowed` sounds fine to me, unless we also record policy decisions (policy_request_allowed)?

Reply:
`request_allowed` is good IMHO.